ENIGMAsystem/SHELL/Audio_Systems/DirectSound/DSsystem.h

```diff
@@ -21,7 +21,10 @@
 #include <dsound.h>
 #include <mmsystem.h>
 #include <stddef.h>
+#define byte __windows_byte_workaround
 #include <windows.h>
+#undef byte
```

Comment:
What's really going on here is `#include <windows.h>` needs to go before `using namespace std;` to get rid of these errors/warnings. This is just putting a bandaid over the real problem, which is pollution of the global namespace.